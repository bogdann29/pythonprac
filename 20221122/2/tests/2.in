Â íàñòîÿùåå âðåìÿ ïîääåðæêà êîììåð÷åñêèõ øðèôòîâ âûäåëåíà â ñàìîñòîÿòåëüíóþ êîëëåêöèþ PSNFSSx. Ñðåäè ïàêåòîâ PSNFSSx ñëåäóåò âûäåëèòü ïàêåòûhvmaths, lucidabr, ly1, mathtime, mtpro, tmmaths, êîòîðûå ïðåäíàçíà÷åíû äëÿ çàìåíûìàòåìàòè÷åñêèõ øðèôòîâ Computer Modern íà ñîâðåìåííûå øðèôòû Lucida Bright èMathTime, ïðîäàâàåìûå êîìïàíèÿìè Y&Y 9 è MicroPress10 . Ìû ñîñðåäîòî÷èì âíèìàíèå ×èòàòåëÿ íà íåêîììåð÷åñêèõ øðèôòàõ, ïîääåðæèâàåìûõ ïàêåòàìè PSNFSS, òåìáîëåå ÷òî çäåñü òàêæå åñòü çàìå÷àòåëüíûå øðèôòû ñåìåéñòâ Palatino è Times äëÿ ìàòåìàòè÷åñêèõ ôîðìóë.