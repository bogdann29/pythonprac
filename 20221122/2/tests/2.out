В настоящее время поддержка коммерческих шрифтов выделена в самостоятельную коллекцию PSNFSSx. Среди пакетов PSNFSSx следует выделить пакетыhvmaths, lucidabr, ly1, mathtime, mtpro, tmmaths, которые предназначены для заменыматематических шрифтов Computer Modern на современные шрифты Lucida Bright иMathTime, продаваемые компаниями Y&Y 9 и MicroPress10 . Мы сосредоточим внимание Читателя на некоммерческих шрифтах, поддерживаемых пакетами PSNFSS, темболее что здесь также есть замечательные шрифты семейств Palatino и Times для математических формул.
